filters/auth/tokeninfo.go

```diff
@@ -353,6 +362,14 @@ func (f *tokeninfoFilter) Request(ctx filters.FilterContext) {
 	}
 
 	authorized(ctx, uid)
+
+	if f.maskOAuthRealms != nil {
+		realm, ok := authMap[realmKey].(string)
+		if ok && f.maskOAuthRealms.MatchString(realm) {
+			ctx.StateBag()[logfilter.MaskedAuthUserKey] = realm
```

Comment:
Should it rather be `uid` (not `realm`) that is stored under `MaskedAuthUserKey`?

Reply:
idea was to at least see which realm the caller came from.